book/chapter-00.md

```diff
@@ -58,21 +58,21 @@ Example of the expected reward for each miner depending on which chain they work
+*How does this guarantee information security?* Let’s say an attacker wanted to go back and edit the history of the ledger to exclude a certain transaction. The attacker has to go all the way back to the block that precedes the block containing the targeted transaction, fork the blockchain, and mine enough blocks to the point where the length of the revised chain surpasses that of the original chain (transforming the revised chain into the new main, or defacto, chain). In order for the revised chain to surpass the current main chain, the attacker would need a majority of the network’s hashpower. This attack is aptly named the *51% attack*. The deeper a transaction is nested into the history of the blockchain, the more costly and difficult it would be to revise it. The mining setup for Bitcoin and the costs of electricity and computation are substantial in order to prevent this type of attacks.
 
-Let’s think about those tools in our toolbox again. Without hashing, digital signatures, and other components of public key cryptography, we wouldn’t have the guarantee that users own and control their money, and can only send and spend what they own… or that the money can’t be duplicated! Without the difficulty of mining, we wouldn’t have the economic guarantee that what is written to the blockchain will most likely remain there! Wow cryptoeconomics!
+Let’s think about those tools in our toolbox again. Without hashing, digital signatures, and other components of public key cryptography, we wouldn’t have the guarantee that users own and control their money and can only send and spend what they own… or that the money can’t be duplicated! Without the difficulty of mining, we wouldn’t have the economic guarantee that what is written to the blockchain will most likely remain there! Wow cryptoeconomics!
 
 ## PROBLEMS
 
 ```
-Q) Why do we need Decentralized Consensus not just decentralized code verification and execution?
+Q) Why do we need Decentralized Consensus and not just decentralized code verification and execution?
 A) To prevent the double spend attack.
-Q) If you are a miner will the block you mine in POW always be included in the longest chain?
+Q) If you are a miner, will the block you mine in PoW always be included in the longest chain?
 A) No! Because of forks.
-Q) How can you make it harder for someone to attack the altcoin paypal chain?
```

Comment:
re - "Q) How can you make it harder for someone to attack the altcoin paypal chain?
A) Sharding - make attacks all or nothing and Plasma "

(1) This one doesn't really seem to be answerable from the info in the chapter (I'm assuming that's the idea of the Problems); and (2) "altcoin PayPal chain isn't defined above; "PayPal" is mentioned in the context of what we'll be learning ("Web 2.0 PayPal" and "Web 3.0 PayPal), but there's nothing that indicates to the reader what's meant by "altcoin PayPal chain."